profiles/common/modules/custom/nexteuropa_varnish/nexteuropa_varnish.module

```diff
@@ -223,6 +223,110 @@ function nexteuropa_varnish_node_presave($node) {
   }
 }
 
+/**
+ * Implements hook_node_delete().
+ */
+function nexteuropa_varnish_node_delete($node) {
+  // No rule defined for the node: Ignore the rest of the process.
+  if (!nexteuropa_varnish_node_has_rules($node)) {
+    return;
+  }
+
+  $paths = array();
+
+  $entity_uri = entity_uri('node', $node);
+
+  if ($entity_uri) {
+    $url_locale = language_negotiation_get('language', 'locale-url');
+    $url_suffix = language_negotiation_get('language', 'nexteuropa_multilingual_url_suffix');
+    // Add logic to send a language prefix/suffix regex for non aliased paths.
+    if ($url_locale) {
+      $lgregex = '[a-z]{2}\/';
+    }
+    elseif ($url_suffix) {
+      $lgregex = '_[a-z]{2}';
+    }
+    else {
+      $lgregex = '';
```

Comment:
To decrease the cyclomatic complexity, the `else` condition could be removed here.
```php
$lgrexex = '';

if ($url_locale) {
 // stuff
} elseif ($url_suffix) {
 // stuff
}
```

Reply:
fixed.